### DISCUSSION TITLE
New ideas and suggestions for Anura3D v 2024

### DISCUSSION THREAD

**Discussion by ayerrocolom** [Anura3D/Anura3D_OpenSource#112] [Ideas]
What features or improvements would you like in the new Anura3D version?
Summarize your ideas or suggestions for Anura3D v 2024
Thanks!

**Comment by EmreDumlu-UM** [comment]
We are interested in free surface flows during dam failure due to overtopping. However, we perform our overtopping cases by defining a high storage water level, which gives not a constant water level.  It is an important factor to define constant inflow during this type of simulation. For this reason, I would like to ask for your guidance on defining inflow/outflow BCs or request possible cooperation in ANURA3D.  

Best regards,

**Reply by murphy-john** [reply]
Hi @Emre-ncche, this is a good idea. We'll try to get the inflows/outflows implemented.

**Comment by vjagodnik** [comment]
What about the application of MPM on rockfalls?
I assume it would be a computing challenge.
And micropolar continuum approach.

**Reply by ayerrocolom** [reply]
Thank you for your suggestions! We agree that rockfalls is a great application, however it might be better suited for a DEM type of approach. since this is a more discrete problem rather than a continuum. In any case, we will think about it!

**Comment by qizih** [comment]
We expect that the 3D model of rainfall landslides can define soil surface and hydraulic boundary conditions.

**Reply by ayerrocolom** [reply]
Thanks for your feedback!
Anura3D is prepared to handle 3D meshes already. In order to have a realistic topography, you can create the 3D mesh in GiD. 
However, the calculation will take a long time depending on the size of the mesh. In terms on initial boundary conditions (e.g., hydraulic head), the code needs to be validated for 3D conditions, indeed. We will consider your comments in the future.

**Comment by WHUT-HE** [comment]
Hello, my research direction is earthquake induced landslide instability. As a beginner, I am still very confused about the input of seismic waves and do not have much information to refer to. I hope that future versions can simplify the input of seismic waves.

**Reply by vjagodnik** [reply]
I think there is a paper regarding the use of Anura3D on seismic actions. If I am right, @ayerrocolom it the main author.

**Reply by WHUT-HE** [reply]
Thank you very much. I have found this paper, but it is still too difficult for me to reproduce. I hope the next version of the user manual can include an earthquake related case analysis。

**Reply by kuailebozai** [reply]
Hi, is there any progress in the research about the seismic direction, I also want to use Anura3d to simulate the analysis of the effect of seismic loads on the stability of the structure, but I don't know how to apply the seismic loads, if you find a way please contact me, thank you very much! My e-mail: kaiboyang2023@163.com.

**Comment by lurpi** [comment]
How strong is the requirement to have Anoura only compiled in windows? what are the roadblocks to having it compiled under other OS?

**Comment by ZhuhaoGu** [comment]
I think we can consider developing a function that can produce cracks, if you can introduce the crack function after the collision, it is a good choice!
